src/styles/sidebar/_base.scss

```diff
@@ -0,0 +1,37 @@
+@use 'tailwindcss/base';
+
+@layer base {
+  html,
+  body {
+    @apply h-full;
+  }
+
+  body {
+    @apply bg-white font-sans text-base text-color-text;
+  }
+
+  // TODO: Find all component-src uses of <a> elements directly and update to
+  // use `Link` component (for styling) or verify that they don't rely on this
+  // element style
+  a {
+    // Ensure the tag has width in order for :focus styling to render correctly.
+    @apply inline-block;
+    // Give links a radius to allow :focus styling to appear similar to that of buttons
+    @apply rounded-sm;
+    @apply text-brand no-underline;
+  }
+
+  a:active,
+  a:focus {
+    @apply no-underline;
+  }
+
+  a:hover {
+    @apply text-brand-dark no-underline;
+  }
+
+  // TODO: Research history of this and whether it's still appropriate
```

Comment:
Searching through commits with `git log -p -G '\bhyphens\b'` it seems that it dates back to at least 2013, and there was no detailed explanation. It might simply be part of an old reset. I'd suggest to make a "from scratch" decision on whether to keep this or not.

Reply:
I'm going to review this rule when I sweep back through these elements styles during the assessment of the anchor tag rules in here, and at that time look at it more closely and decide whether it makes sense to keep. _Probably_ not.